trueline.sh

```diff
@@ -116,6 +116,24 @@ _trueline_venv_segment() {
     fi
 }
 
+_trueline_has_conda_env() {
+    printf "%s" "${CONDA_DEFAULT_ENV}"
+}
+
```

Comment:
Remove this line (the idea is to keep the same style we have with the venv)

Reply:
removed the line